Documentation/watch-recovery.md

```diff
@@ -0,0 +1,81 @@
+
+Open a terminal and a browser window with Tectonic Console. Keep them open side by side for this exercise. Kubernetes will rebuild the killed containers quickly.
+
+First, use Tectonic Console to determine which nodes are running the API server.
+Go to *Workloads > Pods*, and enter ‘api’ in the search field.
+
+Then, select one of the nodes listed, and SSH to its IP address.
+
+```
+Sssh core@ip
+```
+
+Remember that the kubelet runs on **reconciliation loops** which are constantly work to keep the system in a stable state. When we execute the command to kill this API server, the kubelet will notice immediately.
+
+Test this by killing the active containers running the Kubernetes API server on this node. Because the API server is a DaemonSet, it should be running on all of the master nodes.
+
+From one of the masters, get the IDs of the containers running the API Server:
+
+```
+$ docker ps -f 'name=.*kube-apiserver.*' --format "{{.ID}}"
+4ae686927f22
+06bb5ba95034
+```
+Then, kill the listed containers.
+
+```
+$ docker kill 4ae686927f22 06bb5ba95034
+```
+
+Within a few seconds the kubelet will notice that the API server Pod is no longer running, and will relaunch under a new ID:
+
+```
+$ docker ps -f 'name=.*kube-apiserver.*' --format "{{.ID}}"
+f4a262619d1d
+d964fa94c69b
+```
+
+In the console, watch the list  of API servers to see that one is disabled, and a new one is launched. Note that the Tectonic Console session remained working because the API is configured with High Availability. This is the same Kubernetes features that enable your apps to remain up and running without constant developer monitoring and interaction.
+
+## Simulate Node failure
+
+Killing the API server acts as an example of an individual process failure. Next, simulate a power down or network partition by disabling an entire node.
+
+Kubelet uses reconciliation loops to remain in constant contact with the API server and report back status. Use iptables to create a temporary firewall rule and simulate an interruption in this connection.
+
+```
+$ iptables -A OUTPUT -p tcp --dport 443  -j DROP
+$ systemctl stop dockerk
+$ systemctl stop kubelet
+$ ps aux | grep kube
+```
+
+Use Tectonic Console to inspect the node. Notice that the node is marked ‘unhealthy’, and nothing is rescheduled.
+
+Check your app. Even though the node is unhealthy, and the rescheduling process has not yet begun, the app is still up and running as before. Because the cluster runs in High Availability, the app does not go down, and your clients experience no interruption in service. Kubernetes maintains persistent availability automatically.
+
+Wait 5 mins, then reinspect Console. Notice that new nodes have been scheduled to repair the interruption.
+
+Tectonic configures the kubelet to wait 5 minutes after a node is determined to be unhealthy before workloads are moved off the node. Because nodes are run in High Availability, redundancy allows the system this time to regroup.
```

Comment:
"...redundancy allows the system this time to regroup." <-- Why say "this time"? Does this not consistently work?